iModelCore/ECDb/Tests/NonPublished/ECSqlStatementTests.cpp

```diff
@@ -9017,6 +9017,104 @@ TEST_F(ECSqlStatementTestFixture, OrderBy)
     ASSERT_EQ(0, strcmp(lastName, "Da Vinci"));
     }
 
+
+//---------------------------------------------------------------------------------------
+// @bsimethod
+//---------------------------------------------------------------------------------------
+
+TEST_F(ECSqlStatementTestFixture, NullsOrdering)
+{
+    // Create StartupCompany
+    ASSERT_EQ(BentleyStatus::SUCCESS, SetupECDb("StartupCompany.ecdb", SchemaItem::CreateForFile("StartupCompany.02.00.00.ecschema.xml")));

```

Comment:
Do not use the outdated StartupCompany schema for new tests. It's an EC3.1 schema. And unnecessarily complex.
Just import a new schema with a single class and a single property like we discussed earlier. Examples are in SchemaRemapTests.

Reply:
yes, But I am using the older one just to understand the existing flow of the tests. I have plans to come up with a new test where I will use schema 3.0.